org.eclipse.january/src/org/eclipse/january/dataset/SliceND.java

```diff
@@ -45,19 +45,19 @@ public SliceND(final int[] shape) {
 	}
 
 	/**
-	 * Construct ND slice from an array of 1D slices
-	 * @param shape
-	 * @param slice
+	 * Constructs a SliceND Object from an array of 1D slices
+	 * @param shape Array of shapes
+	 * @param slice Slices array to add inside of the SliceND
```

Comment:
should be `@param slice Slice for each dimension of ND slice` (actually improvements on wording welcome, but the idea is that each entry in the array applies to a dimension is important.